wrappers/src/list_cs.cpp

```diff
@@ -90,6 +83,31 @@ REALM_EXPORT Object* list_set_embedded(List& list, size_t list_ndx, NativeExcept
     });
 }
 
+REALM_EXPORT void* list_set_collection(List& list, size_t list_ndx, realm_value_type type, NativeException::Marshallable& ex)
```

Comment:
Same comment - feels like there's some code duplication between set and insert that we can avoid.